components/Builder/Builder.tsx

```diff
@@ -1,20 +1,24 @@
-import { Tabs } from "components";
 import { SectionHeader } from "components/SectionHeader/SectionHeader";
 import { BaseOuterWrapper } from "components/style/Wrappers";
 import { laptopAndUnder, mobileAndUnder, tabletAndUnder } from "constant";
+import { useInView } from "framer-motion";
 import { useAddHashToUrl } from "hooks/helpers/useAddHashToUrl";
+import dynamic from "next/dynamic";
 import OO from "public/assets/oo-logo.svg";
 import { useRef } from "react";
 import styled from "styled-components";
 
+const Tabs = dynamic(() => import("components/Tabs/Tabs"));
```

Comment:
we're doing a whole lot of this now. https://nextjs.org/learn/seo/improve/dynamic-imports